test/groovy/util/FixtureHelper.groovy

```diff
@@ -715,11 +716,490 @@ class FixtureHelper {
         )
     }
 
+    static Map createSockShopTestResults() {
+        return JUnitParser.parseJUnitXML(
+            createSockShopJUnitXmlTestResults()
+        )
+    }
+
     static Set createTestResultErrors() {
         return JUnitParser.Helper.getErrors(createTestResults())
     }
 
+    static Set createSockShopTestResultErrors() {
+        return JUnitParser.Helper.getErrors(createSockShopTestResults())
+    }
+
     static Set createTestResultFailures() {
         return JUnitParser.Helper.getFailures(createTestResults())
     }
+
+    static Set createSockShopTestResultFailures() {
+        return JUnitParser.Helper.getFailures(createSockShopTestResults())
+    }
+
+    static List createIssuesForJQLQuery() {
```

Comment:
@georgfedermann I strongly suggest not to add such non-parametrizable to the fixture. The nature of this implies that it is needed in a specific scenario only. In this case, please move directly into the test.

Reply:
@georgfedermann also, note that you call this method in the context of `getIssuesForJQLQuery`. This function executes a random JQL query. What may work in your scenario now, may not be in line with the expectations of somebody else. I, for example, use this function within `Project` to query issues of type bug. Here, this would return some unrelated test case. Needs to be reworked.